.codegen/api.java.tmpl

```diff
@@ -131,7 +131,18 @@ public class {{.PascalName}}API {
 {{define "method-call-paginated" -}}
   {{- if .Pagination.MultiRequest -}}
     {{- if and .Pagination.Offset (not (eq .Path "/api/2.0/clusters/events")) -}}
-    request.set{{.Pagination.Offset.PascalName}}({{if eq .Pagination.Increment 1}}1{{else}}0{{end}}L);{{end -}}
+    request.set{{.Pagination.Offset.PascalName}}(
+    {{- if eq .Pagination.Increment 1 -}}
+      1
+    {{- else if contains .Path "/scim/v2/" -}}
+      1
+    {{- else -}}
+      0
+    {{- end}}L);{{end -}}
+    {{if and .Pagination.Limit (contains .Path "/scim/v2/")}}
+    if (request.get{{.Pagination.Limit.PascalName}}() == 0) {
```

Comment:
nit: 0L instead of 0. It doesn't matter because implicitly java will know 0 is of Long type. But mentioning since we specify 1L and 100L below so might be good for consistency on code reading purposes.